src/scripts/game.coffee

```diff
@@ -51,25 +54,26 @@ class Game
   landPlane: (plane) ->
     # @todo flight ordering on landing
     return unless plane.isFlying()
+    return unless route = Route.find(key: plane.location)
 
     plane.location = plane.location.split('->')[1]
-    @processRewardsFor plane
+    @processRewardsFor plane, route
     MessageBus.publish 'dataChange'
 
   buyPlane: (player) ->
     Plane.createFor( player )
-    player.money -= 300
+    player.money -= PLANE_COST
     MessageBus.publish 'dataChange'
 
-  processRewardsFor: (plane) =>
-
+  processRewardsFor: (plane, route) =>
+    return if @_stop
```

Comment:
I noticed that there is a race cond when winning / losing the game... if flights take off between MessageBus updates - you can win a game, and as the You win dialog is being rendered - pink planes land and retake locations.... making it very confusing to the user who thinks... 'oh cool I won' - then see's the newly acquired pink locations, and wonders - how did I win (i.e. Arek's experience)?